install_config/persistent_storage/persistent_storage_azure_file.adoc

```diff
@@ -42,6 +42,11 @@ $ /usr/sbin/setsebool -P virt_use_samba on
 $ /usr/sbin/setsebool -P virt_sandbox_use_samba on
 ----
 
+[NOTE]
+====
+Mount options is not available in {product-title} 3.7.43. You should not use Azure File in {product-title} 3.7.43. You can upgrade to {product-title} 3.7.46 or later to use Azure File. 
```

Comment:
s/is/are
s/should not/cannot
I'd say "To use Azure File, upgrade to {product-title} 3.7.46 or later."

I'd double-check the conditionalization of this note.